### PR TITLE
[Feature] Add vols to node settings [OSF-8059]

### PR DESCRIPTION
**Pull request by fabmiz**
## Purpose

Add view-only links to node/project settings

## Changes

- Remove private link and responsive table styles	
- Use manageContributors only on contributors page
- Remove private links table html
- Add vols to node settings page
- Create a separate template for vols
- Add css styles for private links template

## Before

screen shot 2017-07-05 at 11 48 03 am

## After

screen shot 2017-07-05 at 11 47 45 am

## Ticket

https://openscience.atlassian.net/browse/OSF-8059

### CODE REVIEW

File: website/templates/project/settings.mako [hunk 1]
```diff
@@ -186,6 +189,26 @@
 
         % endif  ## End Select Addons
 
+        % if 'admin' in user['permissions']:  ## Begin create VOLS
+            % if not node['is_registration']:
+            <div class="panel panel-default">
```
Comment (alexschiller) [inline website/templates/project/settings.mako]: Minor: indent

File: website/templates/project/settings.mako [hunk 2]
```diff
@@ -474,6 +497,7 @@
       window.contextVars.wiki.isEnabled = ${wiki.short_name in addons_enabled | sjson, n };
       window.contextVars.currentUser = window.contextVars.currentUser || {};
       window.contextVars.currentUser.institutions = ${ user['institutions'] | sjson, n };
+      window.contextVars.user = ${ user | sjson, n };
```
Comment (alexschiller) [inline website/templates/project/settings.mako]: What info does this have that isn't available through currentUser?

File: website/templates/project/private_links.mako [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<%include file="modal_generate_private_link.mako"/>
+<link rel="stylesheet" href="/static/css/private_links_table.css">
+<div class="scripted" id="linkScope">
+    <table id="privateLinkTable" class="table responsive-table responsive-table-xs"
```
Comment (alexschiller) [inline website/templates/project/private_links.mako]: screen shot 2017-06-15 at 10 36 01 am
The table is not adequately responsive on the node settings page. Works fine on the contrib page.

File: website/templates/project/settings.mako (hunk 2 above)
Comment (fabmiz) [inline website/templates/project/settings.mako]: In this context, there is no difference. However, sharing.js, which is used both for contributor and private links, expects a user to be set in the context vars,
https://github.com/CenterForOpenScience/osf.io/pull/7335/files#diff-fd51466e5d40043598e7fe4e21bd0100
I can simply set window.contextVars.user = window.contextVars.currentIUser if that's better

File: website/templates/project/private_links.mako (hunk 3 above)
Comment (fabmiz) [inline website/templates/project/private_links.mako]: Fixed.

File: website/templates/project/settings.mako (hunk 2 above)
Comment (fabmiz) [inline website/templates/project/settings.mako]: I was wrong, ctx.currentUser has less info than ctx.user. In our case, currentUser does not have permissions info, which sharing-page needs.

ctx.currentUser:
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/project/project_base.mako#L68

ctx.user:
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/project/views/node.py#L679

Comment (alexschiller) [inline website/templates/project/settings.mako]: That's fine then. Keep as is.

Comment (fabmiz) [inline website/templates/project/settings.mako]: So I will just add permissions info as such `window.contextVars.currentUser.permissions = ${ user['permissions'] | sjson, n } ;`

File: website/templates/project/settings.mako (hunk 1 above)
Comment (alexschiller) [inline website/templates/project/settings.mako]: @fabmiz did this one get missed?

**Comment by brianjgeiger** [comment]
@fabmiz Could you update to fix the merge conflicts? Also, for tickets that have changed UI, it would be awesome to include the before and after screenshots on the PR Description before putting it into code review. I've fixed it for this one, but in the future (and if you have other tickets open now). It's convenient not to have to look through the jira comments to see what the current state of the screenshots are.

**Pull request merged**